kinds/dashboard/dashboard_kind.cue

```diff
@@ -353,7 +361,7 @@ lineage: schemas: [{
 		#Threshold: {
 			// Value represents a specified metric for the threshold, which triggers a visual change in the dashboard when this value is met or exceeded.
 			// Nulls currently appear here when serializing -Infinity to JSON.
-			value: number | null @grafanamaturity(NeedsExpertReview)
+			value?: number | null @grafanamaturity(NeedsExpertReview)
```

Comment:
This field has given us no end of trouble, and here it is again!

There were a bunch of dashboards where this field was absent. I'm not sure if that's a result of running that stripnulls script (which should no longer be necessary now), but this is a case where AIUI `null` is semantically different than field absence, so i thought making it optional was the right course of action.